charts/openstack-cloud-controller-manager/values.yaml

```diff
@@ -46,9 +46,10 @@ livenessProbe: {}
 # Set readinessProbe in the same way like livenessProbe
 readinessProbe: {}
 
-# Set nodeSelector where the controller should run, i.e. controlplane nodes
```

Comment:
you could still override the values using helm, no matter what is defined in this file